website/content/api-docs/system/internal-counters.mdx

```diff
@@ -347,6 +349,7 @@ This endpoint was added in Vault 1.6.
 
 - `start_time` `(string, optional)` - An RFC3339 timestamp or Unix epoch time. Specifies the start of the
   period for which client counts will be reported. If no start time is specified, the billing start date will be used.
+  The billing start date is the auto-rolled start date to the most recent year. 
```

Comment:
I think if I read about this field right here, not seeing any of the other pages, I would be a bit confused what this means. Maybe adding a link to the example page here would be helpful.

Reply:
Agreed, I'm confused about what "auto-rolled start date" means in relation to the most recent year. Also, is "most recent year" the most recent calendar year? The most recent billing year (if the 2 are not the same)?

Reply:
It would be the most recent billing year which would also be the recent calendar year. 
Would it be better to have "The billing start date is the auto-rolled start date to the most calendar (billing) year. Please refer to upgrade [Upgrade docs link] for more information"?